docs/installation/prepare/jetson-agx-orin-setup.md

```diff
@@ -82,13 +82,24 @@ If this command fails, try using `wget https://storage.googleapis.com/packages.v
 
 ## Serial Communication Protocol Tips
 
+To change the pins that are in use for modes of serial communication, launch <file>jetson-io.py</file> with the following command:
+
+```sh { class="command-line" data-prompt="$"}
+sudo /opt/nvidia/jetson-io/jetson-io.py
```

Comment:
Intuitively, this should be all you need. In practice, though, I've found that it only works when you have write access to the current working directory. So, `cd ~; sudo /opt...` works, while `cd /sys/class; sudo /opt...` does not. Is that worth documenting? Maybe not. but it caught me off-guard.

Similarly, `jetson-io.py` only displays its menu if your screen is large enough. If your terminal window is very small because you didn't fullscreen it when you started, nothing will be displayed until you resize your window. Is that worth documenting? Again, maybe not, but it surprised me the first time I hit it.

Reply:
Yup, that's great! TBH what you had originally was probably good enough, too, and this is overkill.